tools/oas-to-am/test/features/fixtures/expected.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<AssignMessage name="Assign.createPetsRequest">
+  <AssignTo createNew="true" type="request">createPetsRequest</AssignTo>
+  <AssignVariable>
+    <Name>createPets.url</Name>
+    <Value>https://httpbin.org</Value>
+  </AssignVariable>
+  <Set>
+    <Path>post</Path>
```

Comment:
how about https://docs.apigee.com/api-platform/reference/policies/assign-message-policy#set-verb?

Reply:
in this case, i was setting the basepath (confusing because /post is the path for httpbin!)

Reply:
fixed